### PR TITLE
Update monitoring.md

### PR DESCRIPTION
**Pull request by Rajakavitha1**
Fixes: https://github.com/Mirantis/mke-docs/issues/88

### CODE REVIEW

File: content/docs/operations/monitoring.md [hunk 1]
```diff
@@ -30,14 +30,21 @@ monitoring:
 
 To access the Grafana dashboard:
 
+1. Get the `admin` user password of the Grafana dashboard from the `monitoring-grafana` secret in the `mke` namespace.
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:33]:
```suggestion
1. Obtain the `admin` user password for the Grafana dashboard from the `monitoring-grafana` secret in the `mke` namespace.
```

File: content/docs/operations/monitoring.md [hunk 2]
```diff
@@ -30,14 +30,21 @@ monitoring:
...
+   ```bash
+    kubectl get secret monitoring-grafana -n mke -o jsonpath="{.data.admin-password}" | base64 --decode
+    ```
+   
 1. Run the following command to `port-forward` Grafana:
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:38]:
```suggestion
2. Run the following command to `port-forward` Grafana:
```

File: content/docs/operations/monitoring.md [hunk 3]
```diff
@@ -30,14 +30,21 @@ monitoring:
...
 
     ```bash
     kubectl --namespace mke port-forward svc/monitoring-grafana 3000:80
     ```
 
-2. Go to `http://localhost:3000`.
-
+1. Go to `http://localhost:3000`.
+   **Welcome to Grafana** login page appears.
+   Login using the default username `admin` and provide the password that you retrieved from the `monitoring-grafana` secret.
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:46]:
```suggestion
3. Navigate to `http://localhost:3000` to access the **Welcome to Grafana** login page.

4. Enter the default username **admin** into the **Email or username** field and type the password you retrieved from the `monitoring-grafana` secret into the **Password** field.

5. Click **Log In**.
```

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

File: content/docs/operations/monitoring.md [hunk 4]
```diff
@@ -30,14 +30,23 @@ monitoring:
 
 To access the Grafana dashboard:
 
-1. Run the following command to `port-forward` Grafana:
+1. Obtain the `admin` user password for the Grafana dashboard from the `monitoring-grafana` secret in the `mke` namespace.
+   ```bash
+    kubectl get secret monitoring-grafana -n mke -o jsonpath="{.data.admin-password}" | base64 --decode
+    ```
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:36]:
```suggestion
   ```bash
   kubectl get secret monitoring-grafana -n mke -o jsonpath="{.data.admin-password}" | base64 --decode
   ```
```

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

File: content/docs/operations/monitoring.md [hunk 5]
```diff
@@ -30,14 +30,22 @@ monitoring:
 
 To access the Grafana dashboard:
 
-1. Run the following command to `port-forward` Grafana:
+1. Obtain the `admin` user password for the Grafana dashboard from the `monitoring-grafana` secret in the `mke` namespace.
+   ```bash
+   kubectl get secret monitoring-grafana -n mke -o jsonpath="{.data.admin-password}" | base64 --decode
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:35]: Please put a line between the step and the codeblock. Also, the codeblock formatting should be closed with ```. I tried to do show that as a Suggestion, however I could not get it to work. A good reference is the codeblock under step 2.

File: content/docs/operations/monitoring.md [hunk 6]
```diff
@@ -30,14 +30,22 @@ monitoring:
...
+   
+2. Run the following command to `port-forward` Grafana:
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:37]:
```suggestion
2. Port forward Grafana:
```

Comment (KoryKessel-Mirantis) [reply content/docs/operations/monitoring.md:37]: Please change the similar step for Prometheus to read in the same manner.

File: content/docs/operations/monitoring.md [hunk 7]
```diff
@@ -30,14 +30,22 @@ monitoring:
...
 
     ```bash
     kubectl --namespace mke port-forward svc/monitoring-grafana 3000:80
     ```
 
-2. Go to `http://localhost:3000`.
+3. Navigate to `http://localhost:3000` to access the **Welcome to Grafana** login page.
```
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:43]: Please edit the similar step in Prometheus to read the same way. Something like:

"Navigate to <URL> to access the <name of Prometheus entry page> page."

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

File: content/docs/operations/monitoring.md (hunk 5 above)
Comment (KoryKessel-Mirantis) [inline content/docs/operations/monitoring.md:35]:
```suggestion
1. Obtain the `admin` user password for the Grafana dashboard from the `monitoring-grafana` secret in the `mke` namespace.

   ```bash
   kubectl get secret monitoring-grafana -n mke -o jsonpath="{.data.admin-password}" | base64 --decode
   ```
```

**Review by KoryKessel-Mirantis — CHANGES_REQUESTED** [review]

File: content/docs/operations/monitoring.md (hunk 7 above)
Comment (Rajakavitha1) [reply content/docs/operations/monitoring.md:44]: done. at localist:9090 Prometheus displays the status page of Prometheus (https://prometheus.io/docs/prometheus/latest/getting_started/#starting-prometheus).

**Review by KoryKessel-Mirantis — APPROVED** [review]

**Pull request merged**